### PR TITLE
Update websocket docs to describe attribute

### PR DESCRIPTION
**Pull request by raboof**
Refs #3438

### CODE REVIEW

**Review by jrudolph — APPROVED** [review]
LGTM (we might want to change the ordering to high-level first, then low-level to make sure people don't miss it)

File: docs/src/main/paradox/server-side/websocket-support.md [hunk 1]
```diff
@@ -46,28 +46,55 @@ For sending data, you can use @scala[`TextMessage.apply(text: String)`]@java[`Te
...
 @@@
 
-Instead, the synthetic @apidoc[UpgradeToWebSocket] represents a valid WebSocket upgrade request. An application can detect
-a WebSocket upgrade request by looking for the @apidoc[UpgradeToWebSocket] header. It can choose to accept the upgrade and
+See the @github[full routing example](/docs/src/test/java/docs/http/javadsl/server/WebSocketCoreExample.java).
+
+## WebSocketUpgrade
+
+To handle websocket requests, you can either use the @apidoc[WebSocketUpgrade] attribute directly, or
+use the more high-level @ref[Routing Support](#routing-support) described below.
```
Comment (ennru) [inline docs/src/main/paradox/server-side/websocket-support.md:82]:
```suggestion
use the more high-level @ref[Routing Support](#routing-support) described above.
```

**Review by ennru — APPROVED** [review]
LGTM.

File: docs/src/main/paradox/server-side/websocket-support.md (hunk 1 above)
Comment (raboof) [reply docs/src/main/paradox/server-side/websocket-support.md:82]: Heh. I actually planned to remove this paragraph (and forgot), but keeping it like this is even better :+1:

**Pull request merged**